pytensor/link/pytorch/linker.py

```diff
@@ -13,7 +13,12 @@ def input_filter(self, inp: Any) -> Any:
         return pytorch_typify(inp)
 
     def output_filter(self, var: Variable, out: Any) -> Any:
-        return out.cpu()
+        from torch import is_tensor
+
+        if is_tensor(out):
+            return out.cpu()
```

Comment:
This will probably create conflict when one of my other PRs gets merged as an FYI.